packages/sass-variables/_responsive.scss

```diff
@@ -0,0 +1,29 @@
+$breakpoints: (
+  // Portrait-mode phones are generally smaller than the 'small' breakpoint.
+    'small': 576px,
```

Comment:
I grabbed these values from [this article](http://devfacts.com/media-queries-breakpoints-2020/) - they seem to be correct for all the emulatable devices in chrome dev tools, anwyay